src/install.rs

```diff
@@ -104,10 +104,10 @@ fn write_disk(
     // copy the image
     write_image(source, dest, true)?;
     reread_partition_table(dest)?;
-    udev_settle()?;
 
     // postprocess
     if ignition.is_some() || config.firstboot_kargs.is_some() || config.platform.is_some() {
+        udev_settle_until_partition_appears(&config.device, "boot")?;
```

Comment:
Let's not hardcode the partition label both inline in `blockdev.rs` and inline here.